text_formatter.go

```diff
@@ -61,7 +65,16 @@ type TextFormatter struct {
 
 func (f *TextFormatter) init(entry *Entry) {
 	if entry.Logger != nil {
-		f.isTerminal = IsTerminal(entry.Logger.Out)
+		f.isTerminal = f.checkIfTerminal(entry.Logger.Out)
+	}
+}
+
+func (f *TextFormatter) checkIfTerminal(w io.Writer) bool {
+	switch v := w.(type) {
+	case *os.File:
```

Comment:
[it should be, yes](https://golang.org/pkg/os/#pkg-variables). But this hasn't changed. We're still checking the type the exact same way we were before.